scripts/falco-driver-loader

```diff
@@ -169,10 +169,15 @@ get_target_id() {
 		# 5.15.0-1009-aws -> ubuntu-aws
 		if [[ $KERNEL_RELEASE =~ -([a-zA-Z]+)(-.*)?$ ]];
 		then
-			TARGET_ID="ubuntu-${BASH_REMATCH[1]}${BASH_REMATCH[2]}"
+			TARGET_ID="ubuntu-${BASH_REMATCH[1]}"
 		else
 			TARGET_ID="ubuntu-generic"
 		fi
+
+		if [[ $(uname -v) =~ (^\#[0-9]+\~[^-]*-Ubuntu .*$) ]];
+		then
+			KERNEL_VERSION=$(uname -v | sed 's/#\([^-\\ ]*\).*/\1/g')
```

Comment:
In case kernel-version is like: 
```
uname -v
#27~22.04.1-Ubuntu SMP Wed May 24 12:50:32 UTC 2023
```
we want to use `27~22.04.1` as our kernelversion, not just `27`.